i18n/translations/fr/admin-users.json

```diff
@@ -54,5 +54,8 @@
   },
   "systemAdministration": "Administration du système",
   "accountAdministration": "Administration des comptes",
-  "userAdministration": "Administration des l’utilisateurs"
+  "userAdministration": "Administration des l’utilisateurs",
+  "error": {
+    "permissions": "[FR]Failed to update permissions."
```

Comment:
We should get a translation for this before merging.

Reply:
Sounds good to me :)
@anikbrazeau would it be possible to get a translation for this string "Failed to update permissions."?